models/aws_cloud_cost__overview.sql

```diff
@@ -1,4 +1,4 @@
-with source_report as (
+with source_report as ( 
```

Comment:
Running this model took a few minutes and processed over 400GB of data 😨. I know we are wanting to move away from incremental models unless absolutely necessary, but maybe this is necessary for this model. What are your thoughts?

Reply:
since we're not going incremental, i'm gonna add a `start_date` variable to limit the date range if folks don't care to analyze old data. 

thoughts?

Reply:
@fivetran-jamie I think that's a great idea!